oracle-plugin/src/main/java/io/cdap/plugin/oracle/OracleSource.java

```diff
@@ -153,6 +154,17 @@ public void validate(FailureCollector collector) {
     public String getTransactionIsolationLevel() {
       return connection.getTransactionIsolationLevel();
     }
+
+    @Override
+    protected void validateField(FailureCollector collector,
+                                 Schema.Field field, Schema actualFieldSchema, Schema expectedFieldSchema) {
+      // Validate if the field schema is of type Decimal and the expected is String
+      if (Schema.LogicalType.DECIMAL.equals(expectedFieldSchema.getLogicalType())
+          && actualFieldSchema.getType().equals(Schema.Type.STRING)) {
+        return;
```

Comment:
The comment and the code does not match here: Comment says `expectedSchema` to be `String` and `actualSchema` but we are validating `actualSchema` to be `String`??

Reply:
Updated